### PR TITLE
Change collection.Sequence to collection.abc.Sequence in keras recurrent.py (with python2 switch)

### PR DESCRIPTION
**Pull request by ulf1**
see https://github.com/tensorflow/tensorflow/pull/35895#issuecomment-575684267
@gunan

### CODE REVIEW

File: tensorflow/python/keras/layers/recurrent.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 from __future__ import division
 from __future__ import print_function
 
+import sys
```
Comment (qlzh727) [inline tensorflow/python/keras/layers/recurrent.py:22]: let's use six, which should be the preferred version check for python.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@ulf1 Can you please check @qlzh727's comments and keep us posted. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
